src/controller/org.controller/org.controller.js

```diff
@@ -69,6 +69,53 @@ async function getCna (req, res, next) {
   }
 }
 
+async function getUsers (req, res, next) {
+  try {
+    const shortName = req.ctx.org
+    const cnaShortName = req.ctx.params.shortname
+    const orgRepo = req.ctx.repositories.getOrgRepository()
+    const userRepo = req.ctx.repositories.getUserRepository()
+    const orgUUID = await orgRepo.getOrgUUID(cnaShortName)
+    let returned = false
+
+    if (!orgUUID) {
+      returned = true
+      logger.info(cnaShortName + ' CNA does not exist.')
```

Comment:
Don't use the word CNA, this may be an ADP or anything else.
```suggestion
      logger.info(cnaShortName + ' organization does not exist.')
```